app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -287,7 +289,7 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
                 prepareAndStartDnsVpn()
             }
         } else if (status) {
-            Utilities.showToastInMidLayout(requireContext(), getString(R.string.always_on_rethink_enabled), Toast.LENGTH_SHORT)
+            showStopDialogAlwaysOn()
```

Comment:
What happens when the user clicks on `Start` after having stopped the app with `Always-on VPN` set?

Reply:
No prompt will be shown. It will start the VPN.